frontend/server/src/Controllers/Session.php

```diff
@@ -324,6 +325,19 @@ private static function getCurrentSessionImpl(
             $currentUser = \OmegaUp\DAO\Users::getByPK(
                 $currentIdentity->user_id
             );
+            $pendingDaysForVerification = null;
+            $now = \OmegaUp\Time::get();
+            if (
+                strtotime(
+                    $currentUser->birth_date
+                ) >= strtotime(
+                    '-13 year',
+                    $now
+                ) && !$currentUser->parent_verified
+            ) {
```

Comment:
Here we need validate that the birthdate is not null:
```suggestion
            if (
                !is_null(
                    $currentUser->birth_date
                ) && strtotime(
                    $currentUser->birth_date
                ) >= strtotime(
                    '-13 year',
                    $now
                ) && !$currentUser->parent_verified
            ) {
```